src/Character.ts

```diff
@@ -8,84 +8,26 @@ import {
 } from './Util/PointUtils'
 
 export class Character {
-  readonly stats: Stats = new Stats()
-  readonly bonuses: Bonuses = new Bonuses()
-  private abilityPoints: number = 30
-  private level: number = 1
-  private isCreating: boolean = true
-  private trainingPoints: number = 0
+  readonly stats: Stats
+  readonly bonuses: Bonuses
+  private abilityPoints = 30
+  private trainingPoints = 0
+  private level = 1
+  private isCreating = true
 
-  //region add stats
```

Comment:
Regions weren't doing what I had hoped in any editor. :( The code is also much smaller once the duplication was removed anyhow.